src/test/scala/chisel3/tests/MixedVec.scala

```diff
@@ -0,0 +1,33 @@
+// See README.md for license details.
+
+package chisel3.tests
+
+import chisel3._
+import chisel3.tester._
+import org.scalatest.FreeSpec
+
+
+class VecIO extends Bundle {
+  val x = UInt(5.W)
+}
+
+class UsesVec extends MultiIOModule {
+  val in   = IO(Input(Vec(4, new VecIO)))
```

Comment:
Can you test the Vec functionality orthogonally from the Bundle functionality (which is probably already tested)? Unless you're specifically trying to test the interaction of both?

Reply:
Removed bundle

Reply:
Renamed this to be VecPokeExpectTest